README.md

```diff
@@ -101,6 +101,7 @@ core:
   worker_num: 0 # default worker number is runtime.NumCPU()
   queue_num: 0 # default queue number is 8192
   max_notification: 100
+  max_concurrent_pushes: 100
```

Comment:
This is core config section, but uses only for Ios. Need to move it to `SectionIos` (config, readme, tests, etc)

Reply:
Yup, forgot to highlight that. While iOS is a more reasonable place as it only effects iOS pushes. The type of config to me seem'd to match better with what was already in place in core. Im moving it to the iOS section now